### DISCUSSION TITLE
Indexing of only the changed/newest files

### DISCUSSION THREAD

**Discussion by whowantsmybigdata** [photoprism/photoprism#4110] [General]
I use syncthing to sync my files including fotos/Videos to my server running photoprism (synced into originals folder). As syncthing is working flawlessly for years I wanted to skip importing via webdav when starting to use photoprism. 
I didn't find an option to autoindex without webdav. Only option seems a "docker compose exec photoprism photoprism index --cleanup" manually or via cron??
I don't get what is taking the indexing so long, when e.g. Only 3 fotos changed since the last indexing, it still goes through (log is saying "convert"?!) every picture (~30tsd.) which takes at least 15-20min using quite a lot of ressources with Raspberry pi 4 4GB, full ssd... 
not a huge deal, but it would be nice to have an option to just index the newest files in order to index more often than every few hours. Any ideas/help?

**Comment by graciousgrey** [comment]
If you run the index command without the --force option only changed (updated FileModificationDate) and new files will be indexed the others will be skipped.
